modules/javafx.controls/src/test/java/test/javafx/scene/chart/NumberAxisTest.java

```diff
@@ -257,72 +257,67 @@ public void checkTickUnitPropertyBind() {
     }
 
     @Test
+    @Timeout(value=1000, unit=TimeUnit.MILLISECONDS)
```

Comment:
minor: I think the convention is to have spaces around the parameters,
e.g. `@Timeout(value = 1000, unit = TimeUnit.MILLISECONDS)`
Just scanned the JavaFX code and we seem to be inconsiostent on that, so I will leave that up to do.

Reply:
I don't think we have a (different) format for annotation arguments, so I've used the function arguments format convention here, no spaces.

One point we could use later is to omit TimeUnit and express the value in seconds, but I kept milliseconds for sake of reviewers.

Reply:
@Maran23 is correct about the convention when there is an annotation (or a function call, for that matter) with an embedded parameter assignment. For example, see various `@Deprecated` annotations in the JDK and JavaFX.

However, as he pointed out, we are not consistent on that, so I would leave it as is for this PR to avoid more churn.